app/src/main/java/com/keylesspalace/tusky/fragment/NotificationsFragment.java

```diff
@@ -202,15 +200,15 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
 
         notifications.clear();
         topLoading = false;
-        topFetches = 0;
         bottomLoading = false;
-        bottomFetches = 0;
         bottomId = null;
         topId = null;
 
         ((SimpleItemAnimator) recyclerView.getItemAnimator()).setSupportsChangeAnimations(false);
         setupNothingView();
 
+        sendFetchNotificationsRequest(null, topId, FetchEnd.TOP, -1);
```

Comment:
Why do we have to request it now but we didn't need to before ?

Reply:
because previously it was triggered by the scroll listener (99% sure)